auto_cpufreq/core.py

```diff
@@ -993,10 +944,10 @@ def set_autofreq():
 
     # determine which governor should be used
     if charging():
-        print("Battery is: charging\n")
+        print("Battery: " + str(battery_percentage()) + "%\tcharging\n")
         set_performance()
     else:
-        print("Battery is: discharging\n")
+        print("Battery: " + str(battery_percentage()) + "%\tdischarging\n")
         set_powersave()
```

Comment:
updated to show percentage and charging status

Reply:
Wasn't too share about adding the battery percentage but now that I think about it, this could be a good addition, but let's change how it looks a bit from visual point of view.

Instead of having it like (as it is now):

> Battery: 97%	charging

to have it like:

> Battery is: charging (97%)